esp-hal-embassy/esp_config.yml

```diff
@@ -0,0 +1,45 @@
+crate: esp-hal-embassy
+
+options:
+- name: low-power-wait
+  description: "Enables the lower-power wait if no tasks are ready to run on the
+               thread-mode executor. This allows the MCU to use less power if the workload allows.
+               Recommended for battery-powered systems. May impact analog performance."
+  default:
+    - value: true
+
+- name: timer-queue
+  description: "The flavour of the timer queue provided by this crate. Integrated
+               queues require the `executors` feature to be enabled.</p><p>If you use
+               embassy-executor, the `single-integrated` queue is recommended for ease of use,
+               while the `multiple-integrated` queue is recommended for performance. The
+               `multiple-integrated` option needs one timer per executor.</p><p>The `generic`
+               queue allows using embassy-time without the embassy executors."
+  default:
+    - if: 'cargo_feature("executors") || is_tooling()'
```

Comment:
What is `is_tooling`? It's not explained anywhere, and it seems like all places set it to `false`.

Reply:
A sorry - this will only be `true` when evaluating the config e.g. in esp-config TUI to make the TUI offer feature gated options.

I'm not sure where to explain this - probably the whole config-specification should be explained in detail in the `README.md` document here?

Reply:
thought so but that would need to be very careful in writing the conditions - e.g. currently

```
 - if: 'cargo_feature("executors") || is_tooling()'
      value: '"single-integrated"'
    - if: '!cargo_feature("executors") && !is_tooling()'
      value: '"generic"'
```

So, if `cargo_feature` returns `true` for everything we won't get "generic" in this situation

Reply:
If `is_tooling` returns `true`, you also don't get `generic` 🤔

Reply:
haha - yeah - we found a bug in the config here

I'm very open to naming suggestions - I personally can't tell which of these two is better - t.b.h. both don't sound ideal to me 🤔